spec/unit/module_loader/puppetfile_spec.rb

```diff
@@ -106,10 +106,15 @@
       expect(subject.instance_variable_get("@overrides")[:modules]).to eq({exclude_spec: true})
     end
 
+    it 'should read the `exclude_spec` setting in the module definition and override the overrides' do
+      module_opts = { git: 'git@example.com:puppet/test_module.git', exclude_spec: false }
+      subject.add_module('puppet/test_module', module_opts)
+      expect(subject.modules[0].instance_variable_get("@exclude_spec")).to be false
+    end
+
     it 'should set :spec_deletable to true for modules in the basedir' do
       module_opts = { git: 'git@example.com:puppet/test_module.git' }
       subject.add_module('puppet/test_module', module_opts)
-      expect(subject.modules[0].spec_deletable).to be true
```

Comment:
Did you mean to delete this? If so, why not delete the whole test case?

Reply:
whoops, that shouldn't be deleted.